frontend/src/components/Table.tsx

```diff
@@ -84,15 +85,19 @@ const Table = <T,>({
         {caption && <Caption>{caption}</Caption>}
         <Thead>
           <Tr>
-            {columns.map((col, i) => (
-              <Th
-                key={col.field + i}
-                sort={col.sortable ? sort.getColumnSort(i) : undefined}
-                width={col.width}
-              >
-                {col.label}
-              </Th>
-            ))}
+            {columns.map((col, i) =>
+              col.label ? (
+                <Th
+                  key={col.field + i}
+                  sort={col.sortable ? sort.getColumnSort(i) : undefined}
+                  width={col.width}
+                >
+                  {col.label}
+                </Th>
+              ) : (
+                <Td key={col.field + i} width={col.width} />
```

Comment:
Why it's `Td` here but not `Th`?

Reply:
@DaoDaoNoCode `Th` cells cannot be empty. In certain tables we have an actions column, so the [PF recommendation is to use `Td`](https://www.patternfly.org/v4/components/table#composable-actions) there.